google/cloud/storage/internal/object_streambuf.cc

```diff
@@ -183,8 +181,6 @@ std::streamsize ObjectReadStreambuf::xsgetn(char* s, std::streamsize count) {
   gbump(static_cast<int>(from_internal));
   offset += from_internal;
   if (offset >= count) {
```

Comment:
I don't know what clang-tidy/our style guide thinks... but do you still need the curly braces?